### DISCUSSION TITLE
How to remove custom fields again once added?

### DISCUSSION THREAD

**Discussion by sschuberth** [community/community#6224] [Projects and Issues] [answered]
I cannot seem to find a way to remove a custom field from a project again once added. Is this possible yet?

**Comment by alexey-igrychev** [comment]
+1

**Comment by alexey-igrychev** [comment, marked as answer]
This issue might be closed
![deletion](https://user-images.githubusercontent.com/17360817/137734511-e86db4a3-1776-41fd-bcbf-90b97f3e9baa.png)

**Reply by sschuberth** [reply]
Would be a bit more natural IMO to be able to delete fields from the "Fields" section in the project's settings. But this works ok enough.
